fhir-persistence-schema/src/main/resources/db2/add_any_resource.sql

```diff
@@ -17,7 +17,7 @@
 -- ----------------------------------------------------------------------------
     ( IN p_resource_type                 VARCHAR( 36 OCTETS),
       IN p_logical_id                    VARCHAR(255 OCTETS), 
-      IN p_payload                          BLOB(2147483647),
+      IN p_payload                          BLOB(1M),
```

Comment:
That's definitely 1MiB not 1MB, right?

Reply:
Fixed to be explicit - I was under the impression it was 1MB (all capital)